targets/drive/mobile/main.jsx

```diff
@@ -122,10 +110,9 @@ const renderAppWithPersistedState = (persistedState = {}) => {
       dictRequire={lang => require(`drive/locales/${lang}`)}
     >
       <CozyProvider store={store} client={client}>
-        <Router
+        <DriveMobileRouter
           history={hashHistory}
```

Comment:
I'm not very happy that we pass the `history` prop here and all other props in [DriveMobileRouter](https://github.com/cozy/cozy-drive/pull/530/files#diff-8278b5d865d4e4819d89588ae0ea7788R1)…